python_modules/dagster/dagster/_core/definitions/sensor_definition.py

```diff
@@ -782,22 +782,6 @@ def build_sensor_context(
     check.opt_str_param(cursor, "cursor")
     check.opt_str_param(repository_name, "repository_name")
 
-    # Determine the set of resources to pass by
```

Comment:
This codepath is not used right now, removing so that the resources-to-build logic is only computed in one place (in the GRPC server)

Reply:
I'm still a bit confused here.

Can you provide a concrete example of a code example that would have worked before this change but that will no longer work? Is it just a case where we pass in a partial set of resources and the rest are filled in from the repository level?

Reply:
```python
build_sensor_context(
    instance=instance,
    sensor_name="my_sensor_name"
)
```

This would (1) pull all resources from the instance and (2) figure out which to bind based on the sensor name, by looking up the sensor in the instance. However, we don't use this pattern internally & I think the number of users who would use it is pretty low (relative to `build_sensor_context(..., resources={...})`). I ended up implementing this pattern pretty early on as an alternative to `resources={...}` but found it not very useful (& couldn't implement a corresponding behavior for schedules anyway since schedules don't take the optional `instance`, `schedule_name` inputs).

Reply:
Cool thanks for the explanation.